.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "kibana-proxy"]
 	path = kibana-proxy
 	url = https://github.com/fabric8io/openshift-auth-proxy.git
+[submodule "lib/openshift-ansible"]
```

Comment:
@ewolinetz @richm  I know its just semantic, but do you think this should live at 'hack/lib' or 'hack/lib/testing' instead?